.github/workflows/publish-docker.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish to Dockerhub
+on:
+    workflow_dispatch: {}
+jobs:
+  publish-to-dockerhub:
+    runs-on: ubuntu-18.04
+    if: github.repository_owner == 'Apicurio'
```

Comment:
Not sure what's the requirement here, but this condition would prevent the workflow from running since the repo owner is not Apicurio. This should be removed if the workflow is intended to be triggered

Reply:
This might suggest that a straight merge from upstream to downstream is not appropriate?